packages/sdk-codegen/src/codeGen.ts

```diff
@@ -241,6 +241,9 @@ export interface ICodeGen {
   /** Do type declarations use a class definition */
   useModelClassForTypes: boolean
 
+  /** Increase indent */
```

Comment:
Why did this sneak in?

Reply:
fyi, as a result of this change, `Remember user's SDK language preference` will show up under the `Features` section in `packages/src/sdk-codegen/CHANGELOG.md` on the next release

Reply:
ah. This happened because an annotation in `docDiffUtils.tsx` was failing the build due to the incomplete interface definition. I'll cherry-pick, open another PR and rebase this one on it's once merged. That CHANGELOG definition wouldn't make much sense so should be avoided